src/TropicalGeometry/variety_supertype.jl

```diff
@@ -15,9 +15,37 @@
 ###
 
 abstract type TropicalVarietySupertype{M,EMB} end
-function pm_object(v::TropicalVarietySupertype)
-  return v.polymakeTV
+# function pm_object(v::TropicalVarietySupertype)
+#   return v.polymakeTV
+# end
+
+
+@doc Markdown.doc"""
+    get_unerlying_polyhedral_complex(TV)
```

Comment:
Typo: un*d*erlying

Also, Maybe drop the `get_` ?

Reply:
Uops, thanks!